packages/google_atlas/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: google_atlas
 description: Google Map Provider for Atlas, an extensible map abstraction for Flutter with support for multiple map providers
-version: 0.3.0
+version: 0.5.0
```

Comment:
Why are we skipping 0.4.0?

Reply:
I did this to reflect the version of atlas it's consuming. It seemed like they've been equal in the past. Since it has all the functionality of atlas 0.5.0 it made sene. 

@felangel  if you don't think this is necessary or a good idea I can make it 0.4.0.

Reply:
I think we should avoid skipping versions and just stick to the semantic versioning guidelines 👍